region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -3137,6 +3182,25 @@
       "unlocksDoors": [{"types": ["ammo"], "requires": []}],
       "flashSuitChecked": true
     },
+    {
+      "link": [8, 8],
+      "name": "Gain Flash Suit (Slopespark)",
```

Comment:
This should exit out the left door.  We have to expect they can 18 tile shinecharge at this level.

Reply:
I'm saying this [8,8] is equivalent to the [4, 4] so it should be deleted.
But there is also a missing [8, 1] that could leave through an open door.